daffodil-cli/NOTICE-bin

```diff
@@ -0,0 +1,74 @@
+Apache Daffodil
+Copyright 2018 The Apache Software Foundation
+
+This product includes software developed at
+The Apache Software Foundation (http://www.apache.org/).
+
+Based on source code originally developed by
+- The Univerisity of Illinois National Center for Supercomputing Applications (http://www.ncsa.illinois.edu/)
+- Tresys Technology (http://www.tresys.com/)
+- International Business Machines Corporation (http://www.ibm.com)
+
+The following NOTICE information applies to binary components distributed with this project:
+
+Apache Commons IO
+  Apache Commons IO
+  Copyright 2002-2016 The Apache Software Foundation
+
+  This product includes software developed at
+  The Apache Software Foundation (http://www.apache.org/).
+
+Apache Xerces Java
+  Apache Xerces Java
+  Copyright 1999-2010 The Apache Software Foundation
+
+  This product includes software developed at
+  The Apache Software Foundation (http://www.apache.org/).
+
+  Portions of this software were originally based on the following:
+    - software copyright (c) 1999, IBM Corporation., http://www.ibm.com.
+    - software copyright (c) 1999, Sun Microsystems., http://www.sun.com.
+    - voluntary contributions made by Paul Eng on behalf of the 
+      Apache Software Foundation that were originally developed at iClick, Inc.,
+      software copyright (c) 1999.
+
+JDOM2
+  Copyright (C) 2000-2012 Jason Hunter & Brett McLaughlin.
+
+  All rights reserved.
+
+  This product includes software developed by the
+  JDOM Project (http://www.jdom.org/).
+
+Jansi
+  Copyright (C) 2009, Progress Software Corporation and/or its
+  subsidiaries or affiliates.  All rights reserved.
+
+Jackson JSON processor
+  # Jackson JSON processor
```

Comment:
Where did the text for this come from?  I'm not saying its wrong, just want to make sure its correct.

Reply:
This comes from the META_INF/NOTICE file inside the jackson-core jar file. The file is in their github here: https://github.com/FasterXML/jackson-core/blob/master/src/main/resources/META-INF/NOTICE

Reply:
Ok, looks good then.